### ISSUE TITLE
How do I make a paragraph-by-paragraph new lines?

### ISSUE THREAD

**Issue by yanaleksandrov** [tovic/rich-text-editor#17]
Hi, the editor is now forcibly removing the paragraph tags. How do I disable BR tag wrapping and enable paragraphs?

**Comment by taufik-nurrohman** [comment]
The `<br>` tag will be converted into paragraph sequence automatically in the `<textarea>`. Have you tried to submit it or click the triple dots button?

**Comment by yanaleksandrov** [comment]
Yes, I saw that the general text is framed by the paragraph tag. But if you delete all the text, and then start on the principle, line-enter-line, then the line is moved through br. This can be seen from the screen on the demo site.
![Screenshot_20210426_233441_com android chrome](https://user-images.githubusercontent.com/40837342/116147712-bb4de580-a6e8-11eb-8478-2362c3c9eb77.jpg)

**Comment by yanaleksandrov** [comment]
Ah, I think I understand what's going on. A double line break gives the paragraph tag. Is it possible to disable the br hyphenation, and wrap each new line with the p tag?

**Comment by taufik-nurrohman** [comment]
All I can say is that this current editor version is in a fixed state. I have no plans to provide such a specific option, unless I have decided to do a [refactor](https://github.com/taufik-nurrohman/rich-text-editor/issues/14).
